packages/api-admin/src/lib/components/Table/filters/singleValueFilters/NumberPickerFilter/index.tsx

```diff
@@ -14,7 +14,7 @@ export function NumberPickerFilter({ setSelectedKeys, selectedKeys, confirm, cle
       <InputNumber
         placeholder="Input number"
         size="small"
-        value={selectedKeys[0]}
+        value={typeof selectedKeys[0] === "bigint" ? selectedKeys[0].toString() : selectedKeys[0]}
```

Comment:
Shouldn't `bigIntToString` be used here as well?

Reply:
Also does typeof work in runtime?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof :see_no_evil:

Reply:
I have no idea how I overlooked that :claun:

Reply:
Messed it up with typescript typechecking 🤦